packages/ui/src/components/ChatMessage/ChatContent.native.tsx

```diff
@@ -0,0 +1,351 @@
+import { utils } from '@tloncorp/shared';
+import {
+  Block,
+  Story,
+  VerseBlock,
+  VerseInline,
+  isImage,
+} from '@tloncorp/shared/dist/urbit/channel';
+import {
+  Inline,
+  isBlockCode,
+  isBlockquote,
+  isBold,
+  isBreak,
+  isInlineCode,
+  isItalics,
+  isLink,
+  isShip,
+  isStrikethrough,
+} from '@tloncorp/shared/dist/urbit/content';
+import { AVPlaybackStatus, ResizeMode, Video } from 'expo-av';
+import { ReactElement, useEffect, useRef, useState } from 'react';
+import { Linking } from 'react-native';
+
+import { Image, Text, View, XStack, YStack } from '../../core';
+import { Button } from '../Button';
+import ContactName from '../ContactName';
+
+function ShipMention({ ship }: { ship: string }) {
+  return (
+    <Button
+      // TODO: implement this once we have a profile screen or sheet
+      // onPress={() => naivigate('Profile', { ship })}
+      backgroundColor="$blueSoft"
```

Comment:
Should use `$positiveBackground` here -- in general we should stick to theme colors (under `dark` or `light` in `tamagui.config.ts`, so that dark mode etc. work as expected. If we need a key not in there, we can expand.